src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTypeTest.php

```diff
@@ -358,10 +382,25 @@ public function testThrowExceptionIfDaysIsInvalid()
 
     public function testSetDataWithDifferentNegativeUTCTimezoneDateTime()
     {
-        date_default_timezone_set('Pacific/Tahiti');
+        $form = $this->factory->create('date', null, array(
+            'format' => \IntlDateFormatter::MEDIUM,
+            'model_timezone' => 'America/New_York',
+            'view_timezone' => 'Pacific/Tahiti',
+            'input' => 'string',
+            'widget' => 'single_text',
+        ));
 
+        $form->setData('2010-06-02');
+
+        $this->assertEquals('01.06.2010', $form->getViewData());
```

Comment:
Would you then suggest to remove these tests and the entry I made to the changelog [here](https://github.com/symfony/symfony/pull/13241/files#diff-9db319ade116fe83563d1a5ad84e6b2aR16)?